### PR TITLE
feat(suite): simplify wellcome left-panel

### PR DESCRIPTION
**Pull request by peter-sanderson**
## Description

Simplifies left panel

## Screenshots:

image
image

### CODE REVIEW

File: packages/components/src/components/assets/TrezorLogo/TrezorLogo.tsx [hunk 1]
```diff
@@ -31,22 +31,27 @@ export interface TrezorLogoProps extends ImgHTMLAttributes<HTMLImageElement> {
...
+            height={typeof height === 'number' ? `${height}px` : height}
+            {...rest}
+        >
+            <StyledReactSVG
+                src={LOGOS[type.toUpperCase()]}
+                beforeInjection={(svg: SVGElement) => {
+                    if (typeof height === 'number') {
+                        svg.setAttribute('height', `${height}px`);
+                    }
+                    svg.setAttribute('fill', theme.TYPE_DARK_GREY);
```
Comment (peter-sanderson) [inline packages/components/src/components/assets/TrezorLogo/TrezorLogo.tsx]: without this, the darkmode coloring was not working.

Comment (jvaclavik) [reply packages/components/src/components/assets/TrezorLogo/TrezorLogo.tsx]: What about using some non-legacy color?

Comment (peter-sanderson) [reply packages/components/src/components/assets/TrezorLogo/TrezorLogo.tsx]: Tell me which one it shall be :pray:

Comment (jvaclavik) [reply packages/components/src/components/assets/TrezorLogo/TrezorLogo.tsx]: What about `iconDefault` or `iconSubdued`?

Comment (peter-sanderson) [reply packages/components/src/components/assets/TrezorLogo/TrezorLogo.tsx]: fixed

Comment (peter-sanderson) [reply packages/components/src/components/assets/TrezorLogo/TrezorLogo.tsx]: Thanks!

**Review by jvaclavik — APPROVED** [review]

**Pull request merged**